### ISSUE TITLE
Поддержка Android ADB

### ISSUE THREAD

**Issue by DarkHobbit** [DarkHobbit/doublecontact#5]
Сделать обмен данными с андроид-устройствами через ADB

**Comment by burunduk** [comment]
Информация для размышлений:
Получить книгу:
`adb pull /data/data/com.android.providers.contacts/databases/contacts2.db .`
Но может быть и `contacts.db`
Дальше:
[dump-contacts2db](https://github.com/ehuott/dump-contacts2db)
Но там нет поддержки вытаскивания всех полей (В пулреквестах есть поле BDAY)
На выходе один vcf файл.

**Comment by DarkHobbit** [comment]
Спасибо. С ADB ещё детально не разбирался. Меня больше беспокоит, что для его работы вроде бы как надо включать в телефоне режим отладки...

**Comment by burunduk** [comment]
Да, надо включать, иначе телефон и не отдаст файл contacts2.db.
Разве что встроенный механизм резервных копий, но тут я даже не знакомился с ним.
